src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj

```diff
@@ -7,16 +7,14 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>19.7.0-preview</Version>
+		<Version>19.8.0-preview</Version>
```

Comment:
nit: Have we made any progress towards releasing a stable version yet?

Reply:
That's a question for @MikhailTryakhov

Reply:
@dsgouda we still have lots of issues in swagger so after they'll be fixed our version would be stable. 
Plan to finish it by Build. Hope to finish much earlier